lib/order.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'menu'
+class Order
+  attr_reader :order, :total_prize
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @total_prize = 0
+  end
+
+  def add_chicken
```

Comment:
It feels like `add_chicken`, `add_lamb` and `add_beef` could be combined into one method? Can you look at the differences between each one and abstract that difference? Happy to talk about this in person if you want more guidance!